src/components/07-form/templates/address-form.njk

```diff
@@ -5,8 +5,7 @@
       <label for="mailing-address-1">Street address 1</label>
       <input id="mailing-address-1" name="mailing-address-1" type="text">
 
-      <label for="mailing-address-2" class="usa-input-optional">
-        Street address 2</label>
+      <label for="mailing-address-2">Street address 2 <span class="usa-input-label-helper">(optional)</span></label>
```

Comment:
Do we like this class name?

Reply:
that sounds good to me. I could see wanting to keep it more descriptive like `usa-input-label-required` but there are probably multiple uses here.

Reply:
Could also be `-note`, `-info` - open to something better.

Reply:
i like `input-label-helper`

Reply:
are there distinct styles for helper text for the `required` text and other (non-`required`) helper text?

Reply:
Yes, required is red and helper is gray.

Reply:
Only alternative I can think of would be `input-label-text` to be slightly more descriptive.